app/models/app_config.rb

```diff
@@ -110,6 +110,11 @@ def bugsnag_release_stage(platform)
     pick_bugsnag_release_stage(platform)
   end
 
+  def crashlytics_project(platform)
+    # apps connected through firebase
+    pick_firebase_app_id(platform)
+  end
+
```

Comment:
See earlier comment, I think `crashlytics_project` should only pick specifically around crashlytics and not tie itself to FAD.